nix/pkgs/images/default.nix

```diff
@@ -48,7 +47,7 @@ let
     # The repo chart is already prepared.
     if [[ "$(semver validate ${tag})" == "valid" ]] &&
       [[ ! ${tag} =~ ^(v?[0-9]+\.[0-9]+\.[0-9]+-0-(main|release)-unstable(-[0-9]+){6}-0)$ ]]; then
-      CHART_FILE=build/charts/Chart.yaml build/scripts/helm/update-chart-version.sh --app-version ${tag}
+      CHART_FILE=build/charts/Chart.yaml build/scripts/helm/update-chart-version.sh --app-version ${tag} --chart-version ${tag} --localpv-provisioner-version 4.2.0 --zfs-localpv-version 2.7.1 --lvm-localpv-version 1.6.2 --mayastor-version 2.8.0
```

Comment:
Hmm, is this supposed to be hardcoded?

Reply:
This needs some changes on update-version-on-release.sh. I've commented this block for now.